packages/fxa-profile-server/package.json

```diff
@@ -46,7 +46,7 @@
     "eslint": "^8.18.0",
     "eslint-plugin-fxa": "^2.0.2",
     "insist": "1.0.1",
-    "mkdirp": "0.5.1",
+    "mkdirp": "1.0.4",
```

Comment:
Q: Do we still need this dependency?

Reply:
No, I'm pretty sure it's going to be taken out! Sorry I didn't expect anyone to be looking at this so soon!